pkg/providers/instancetype/suite_test.go

```diff
@@ -179,8 +181,11 @@ var _ = Describe("Instance Types", func() {
 			"topology.ebs.csi.aws.com/zone": "test-zone-1a",
 		}
 
+		expected := v1alpha5.WellKnownLabels.Difference(sets.NewString(
```

Comment:
I don't think we need to remove the WindowsBuild requirement here. I think we should definitely test that label if we are saying that it's a `WellKnownLabel`

Reply:
Well,It's  
v1.LabelOSStable:                 "linux",

If nobody concerns the mixing of Linux with Windows Build together, I can add it back.